sycl/test-e2e/Graph/Explicit/usm_fill.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: level_zero, gpu
+
```

Comment:
Should we add level zero & opencl back here

Reply:
Updated thanks. This was unintentional.